Source/WebCore/dom/InlineStyleSheetOwner.cpp

```diff
@@ -156,36 +153,37 @@ inline bool isValidCSSContentType(const AtomString& type)
 void InlineStyleSheetOwner::createSheet(Element& element, const String& text)
 {
     ASSERT(element.isConnected());
-    Document& document = element.document();
-    if (m_sheet) {
-        if (m_sheet->isLoading() && m_styleScope)
-            m_styleScope->removePendingSheet(element);
+    Ref document = element.document();
```

Comment:
Why not protectedDocument()? I guess that returns a RefPtr?

Reply:
`Ref document = element.document();` is our preferred pattern for local variable. There is no real benefit to calling `protectedDocument()` here? Using `Ref` like I did makes it clear we're dealing with a Ref (a protected document).